modin/core/dataframe/algebra/groupby.py

```diff
@@ -290,6 +290,9 @@ def caller(
             )
         assert axis == 0, "Can only groupby reduce with axis=0"
 
+        if query_compiler._modin_frame.numeric_columns(True) == []:
```

Comment:
```suggestion
        if len(query_compiler._modin_frame.numeric_columns(True)) == 0:
```
If the left operand of such comparison is a numpy array it would try to do an element-wise comparison and it either returns a boolean mask instead of a boolean scalar, throws a warning, or raise an error.
```python
>>> np.array(["a", "b"]) == []
__main__:1: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
False
```

Reply:
`query_compiler._modin_frame.numeric_columns(True)` returns `list` always. But I agree, checking on the length will be better.